### PR TITLE
rename smartapp to simply 'Konnected Security'

### PR DESCRIPTION
**Pull request by heythisisnate**
Here I'm just trying to polish up some of the branding and language. Some feedback was that "Konnected Alarm Panel (Connect)" was too long. I'm proposing to rename the app to just "Konnected Alarm"

### CODE REVIEW

**Comment by copy-ninja** [comment]
So, (Connect) is used to indicate that the SmartApp is a Service Manager type. That's how SmartThings internally named in their examples.

I suppose Konnected Alarm can be confused with Alarm Clocks too, which is one of my ideas for project down the road. 

I'm proposing Konnected Security (Connect). How's that sound?

**Comment by heythisisnate** [comment]
Konnected Security sounds good to to me! But can we drop the _(Connect)_ part? I don't think any end-users even know what a Service Manager app is, so this designation is meaningless to most people. I didn't even know what it meant until you just pointed it out. I don't think it's a requirement, is it?

I'd also like to propose renaming this repo to `konnected-security`. When the git repo shows up in the SmartThings drop-down it only shows the repo name, so this will be good for branding/identification. What do you think?

**Comment by copy-ninja** [comment]
I'm ok with dropping the (Connect) part, but I think when we send for SmartThings review, they're going to want us to add back (Connect) anyway, because without that, there's no indicator that it's a Service Manager. We may need to come out with certain solution that needs to be outside of this Service Manager app. For instance, delayed arming. It'll make sense when we move on to Irrigation/Sprinklers system. You'll need one Service Manager and a separate app to do scheduling / weather sensing.

File: smartapps/konnected-io/konnected-security.src/konnected-security.groovy [hunk 1]
```diff
@@ -194,11 +203,17 @@ def getDeviceIpAndPort(device) {
...
 }
 
-//Device Discovery : Device Type
-def discoveryDeviceType() { return "urn:schemas-konnected-io:device:AlarmPanel:1" }
-//Device Discovery : Send M-Search to multicast
-def discoverySearch() { sendHubCommand(new physicalgraph.device.HubAction("lan discovery ${discoveryDeviceType()}", physicalgraph.device.Protocol.LAN)) }
-//Device Discovery : Subscribe to SSDP events
+// Device Discovery : Device Type
+def discoveryDeviceType() {
+  return "urn:schemas-konnected-io:device:AlarmPanel:1"
```
Comment (copy-ninja) [inline smartapps/konnected-io/konnected-security.src/konnected-security.groovy]: change this to urn:schemas-konnected-io:device:Security:1

File: firmware/1.5.4.1/app/include/user_config.h [hunk 2]
```diff
@@ -97,7 +97,7 @@ extern void luaL_assertfail(const char *file, int line, const char *message);
 #define LUA_PROCESS_LINE_SIG 2
 #define LUA_OPTIMIZE_DEBUG      3
 
-#define ENDUSER_SETUP_AP_SSID "AlarmPanel"
+#define ENDUSER_SETUP_AP_SSID "konnected-io"
```
Comment (copy-ninja) [inline firmware/1.5.4.1/app/include/user_config.h]: I was thinking the discovered list in SmartApp should be consistent with the SSID. Previosly it was AlarmPanel_XXXXXX in both SSID and SmartApp, so if we change the SSID to just konnected-io, we won't get both places to be consistent.

File: firmware/1.5.4.1/app/include/user_config.h [hunk 3]
```diff
@@ -107,8 +107,8 @@ extern void luaL_assertfail(const char *file, int line, const char *message);
  * if WIFI_STA_HOSTNAME_APPEND_MAC defined: Hostname MUST be 26 chars or less, since last 3 octets of MAC address will be appended
  * if defined hostname is invalid: hostname will default to NODE-xxxxxx (xxxxxx being last 3 octets of MAC address)
 */
-#define WIFI_STA_HOSTNAME "AlarmPanel"
-//#define WIFI_STA_HOSTNAME_APPEND_MAC
+#define WIFI_STA_HOSTNAME "konnected-security-"
```
Comment (copy-ninja) [inline firmware/1.5.4.1/app/include/user_config.h]: Hostname is fine without being consistent with SSID. I think in the documentation, we should always send the user to IP address 192.168.4.1 to go to the enduser setup page

File: smartapps/konnected-io/konnected-security.src/konnected-security.groovy (hunk 1 above)
Comment (copy-ninja) [inline smartapps/konnected-io/konnected-security.src/konnected-security.groovy]: also change the value in ssdp.lua too

Comment (heythisisnate) [inline smartapps/konnected-io/konnected-security.src/konnected-security.groovy]: Got it. Makes sense.

File: firmware/1.5.4.1/app/include/user_config.h (hunk 2 above)
Comment (heythisisnate) [inline firmware/1.5.4.1/app/include/user_config.h]: This variable I believe is only for the first time wifi setup. This is the WiFi AP name that the enduser-setup module broadcasts initially when wifi is not configured. I don't think there is an option to append the mac identifier here.

Comment (copy-ninja) [inline firmware/1.5.4.1/app/include/user_config.h]: it automatically add _XXXXXX last 6 mac address. there's no way around this.

File: firmware/1.5.4.1/app/include/user_config.h (hunk 3 above)
Comment (heythisisnate) [inline firmware/1.5.4.1/app/include/user_config.h]: This hostname is just what you will see if you look in your router's list of wifi devices or DHCP leases. It doesn't show in the SmartApp anywhere. The variable below this line appends the mac identifier. I have 4 devices in my house now, so the identifier makes it easier to differentiate them.

Comment (heythisisnate) [inline firmware/1.5.4.1/app/include/user_config.h]: Yes, will add the IP address of the wifi setup to the docs. This was one of the confusing parts with my neighbors setup.

Comment (copy-ninja) [inline firmware/1.5.4.1/app/include/user_config.h]: yea, agree on this change.

File: firmware/1.5.4.1/app/include/user_config.h (hunk 2 above)
Comment (heythisisnate) [inline firmware/1.5.4.1/app/include/user_config.h]: Ok, so if I change this to konnected-security it should be consistent, right?

Comment (copy-ninja) [inline firmware/1.5.4.1/app/include/user_config.h]: yup

**Comment by heythisisnate** [comment]
Ok, I made some updates.
Are there docs about the SmartApp naming conventions or do you know that from experience?

**Comment by heythisisnate** [comment]
Ok, I did some more reading of the ST examples, and added back the _(Connect)_ to the name. It didn't say that that's a rule anywhere, but you're right that all the examples use that name.

**Pull request merged**